### PR TITLE
LRQA-66587

### PR DESCRIPTION
**Pull request by gleicelisbino**
Related Issue: https://issues.liferay.com/browse/LPS-125860

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -4026,6 +4026,95 @@ definition {
...
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName();
+
+		Form.editDescription();
+
+		Form.addNumericField(fieldPositionNumber = "1");
+
+		var fieldNumericField = FormFields.getFieldName();
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:4049]: @gleicelisbino, how about to rename this variable to `numericFieldName` to better reflect its purpose?

**Review by natocesarrego — COMMENTED** [review]
@gleicelisbino and @diogocordeiro, according to @carolmariaabb, behind the scenes, there is a single back-end code which is providing the feature of confirmation field, hence maybe the existing [Poshi test](https://github.com/liferay/liferay-portal/blob/198687010f7b22654fbe5f1c5978af6a9d7255a9/portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase#L4027) which covers the scenario for the Text Field can be enough to make sure that the new feature is working as designed. In light of this, do you guys think that we should add a similar Poshi test to cover the test scenario for the Numeric Field?

Can you please share your thoughts here?

Thanks.

**Pull request closed without merging**